app/models/rules/sealed_bid.rb

```diff
@@ -0,0 +1,53 @@
+module Rules
+  class SealedBid < Rules::Basic
+    def winning_bid
+      return Presenter::Bid::Null.new if auction.available?
```

Comment:
I would not expect an AR model to return a Presenter. Could we have this return a `NullBid` instead?